src/main/scala/io/buoyant/linkerd/RateLimiter.scala

```diff
@@ -0,0 +1,27 @@
+package io.buoyant.linkerd
+
+import java.util.concurrent.atomic.AtomicInteger
```

Comment:
In Linkerd code we use the style of formatting all imports together in alphabetical order with no blank lines in-between